doc/source/user_guide/reshaping.rst

```diff
@@ -728,14 +728,14 @@ Suppose we wanted to pivot ``df`` such that the ``col`` values are columns,
 ``row`` values are the index, and the mean of ``val0`` are the values? In
 particular, the resulting DataFrame should look like:
 
-.. note::
-
-   col   col0   col1   col2   col3  col4
-   row
-   row0  0.77  0.605    NaN  0.860  0.65
-   row2  0.13    NaN  0.395  0.500  0.25
-   row3   NaN  0.310    NaN  0.545   NaN
-   row4   NaN  0.100  0.395  0.760  0.24
+.. code-block:: html
```

Comment:
This should not be highlighted as html. I think you can probably use just `::` instead for having a code block without any highlighting.

Reply:
Hello Joris, when I use without highlighting checks failing on AzureDevops because flake8-rst giving E999 syntax error. You should break a 'space' after "::", but when I break a space the end of line, ci/code_checks.sh gives linting error. The AzureDevops fails again.

Reply:
I think you should remove the `.. code-block:: html` line, and in the previous have `should look like::` (see the double colon).

If that doesn't work, try `.. code-block:: text` or `.. code-block:: none`

Reply:
Okay I will change it to `.. code-block::`
You will see the AzureDevops fail on `flake8-rst`

Reply:
I guess the default for `code-block` is Python, and `flake-rst` is validating the content of the block assuming it's Python code, hence the syntax error.

In any case, can you implement one of the proposed options @oktaysabak please. Not so important to understand the details of the problem, let's simply fix it and get this merged.

Reply:
Yes flake8-rst wants a ' ' blank after `.. code-block:: ` like this. But if I leave a blank end of line; pandas code checker fails.( `ci/code_checks.sh` ) So I will change it to the `.. code-block:: text` than I will test it locally. I hope it will fix.